lib/cli/commands-schema/aws-service.js

```diff
@@ -46,6 +46,31 @@ commands.set('deploy', {
   ],
 });
 
+commands.set('package', {
+  usage: 'Packages a Serverless service',
+  hasAwsExtension: true,
+  options: {
+    package: {
+      usage: 'Output path for the package',
+      shortcut: 'p',
+    },
+    'aws-s3-accelerate': {
```

Comment:
I believe this flag is supported not only for `package`, but also for commands such as `deploy`, which involve uploading artifacts to S3 that can take advantage of acceleration.

Reply:
that's right the idea was to add it to the package command too, both deploy and package have this option.